integration-tests/gatsby-source-wordpress/package.json

```
@@ -4,8 +4,8 @@
   "private": true,
   "description": "A simple Gatsby WP site for running integrations tests",
   "scripts": {
-    "run-jest": "jest --runInBand",
-    "test": "npm run docker-clean-start && npm run run-jest && cross-env WARM_CACHE=true npm run run-jest",
+    "run-jest": "jest --runInBand && WARM_CACHE=true jest --runInBand",
```

Do we need the `--run-in-band` flag twice?

```suggestion
    "run-jest": "jest --runInBand && WARM_CACHE=true jest",
```

---

We do since we're running jest twice 👍 we run it twice but with the WARM_CACHE env var the second time to mutate some CMS data and test the same tests after a delta update

---

Totally missed that :facepalm: